lib/libs/email/content/upload-subsequent-documents/emailTemplates/Waiver1915BCMS.tsx

```diff
@@ -0,0 +1,35 @@
+import { CommonEmailVariables, Events } from "shared-types";
+import {
+  PackageDetails,
+  BasicFooter,
+  Attachments,
+  SubDocHowToAccess,
+} from "../../email-components";
+import { BaseEmailTemplate } from "../../email-templates";
+
+export const WaiversEmailCMS = ({
+  variables,
+}: {
+  variables: Events["UploadSubsequentDocuments"] & CommonEmailVariables;
+}) => {
+  const previewText = `Action required: review new documents for 1915(B) ${variables.id} in OneMAC.`;
+  const heading = `New documents have been submitted for 1915(B) ${variables.id} in OneMAC.`;
+  return (
+    <BaseEmailTemplate
+      previewText={previewText}
+      heading={heading}
```

Comment:
Asharon pointed out a cleaner way to do this which I like which is to just put the template directly into the BaseEmailTemplate as arguments


  return (
    <BaseEmailTemplate
      previewText={`Action required: review new documents for 1915(B) ${variables.id} in OneMAC.`}
      heading={`New documents have been submitted for 1915(B) ${variables.id} in OneMAC.`}